<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1459</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cameraCountComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="resolutionComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="formatComboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="fpsSpinBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="cameraView1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>661</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="cameraView2" native="true">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="camera1ComboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="camera2ComboBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>130</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRecordingButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectDirectoryButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Dir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1459</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
